pkg/config/config_template.yaml

```diff
@@ -1197,7 +1197,7 @@ api_key:
   #         remove_query_string: false
   ##        @param DD_APM_OBFUSCATION_HTTP_REMOVE_PATHS_WITH_DIGITS - boolean - optional
   ##        If enabled, path segments in URLs containing digits are replaced by "?"
-  #         remove_path_with_digits: false
+  #         remove_paths_with_digits: false
```

Comment:
Code declaring this is [here](https://github.com/DataDog/datadog-agent/blob/ba08dfcc0bb4b569fc5a59e65652b46eeeba42a4/pkg/config/setup/apm.go#L40)